README.md

```diff
@@ -13,6 +13,22 @@ It helps define, organize, track and manage work across individuals and teams. F
 
 The same server binary powers both editions.
 
+## Try out Focalboard 
+
+**Focalboard Personal Desktop**
+
+Try out **Focalboard Personal Desktop** by going to the Windows Store or the Apple AppStore, searching for `Focalboard` and installing to run the compiled version locally. 
+
+If you're running a Linux Desktop, [download the latest `focalboard-linux.tar.gz` release](https://github.com/mattermost/focalboard/releases), unpack the `.tar.gz` archive, and open `focalboard-app` from the `focalboard-app` folder.
+
+While we don't yet offer installs of **Focalboard Personal Desktop** outside of Store-based installs, it is in [consideration for the future](https://github.com/mattermost/focalboard/issues/99) (please upvote the ticket if you're interested in this addition). 
```

Comment:
I would add a "for mac and windows" to be consistent with the linux paragraph added above